### PR TITLE
Feature/convert to kotlin

### PR DESCRIPTION
**Pull request by g-barbera**
Beginning to convert the project to Kotlin; started with `UploadedImage.kt` for an example.

### CODE REVIEW

**Comment by g-barbera** [comment]
This is my first commit to a repo so please bear with me lol. I think it'd be cool if this were to be converted to Kotlin. It'd be a whole lot more concise and would also be useful for other developers to learn an up and coming language.

File: build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.2'
+        classpath 'com.android.tools.build:gradle:7.4.2'
```
Comment (shbhmnk) [inline build.gradle:11]: Not sure if this is required for the kotlin conversion. Generally avoid updating any dependencies.

**Review by shbhmnk — COMMENTED** [review]
Could you link this to an issue/feature ticket. If there is no existing ticket then feel free to create one

File: build.gradle (hunk 1 above)
Comment (g-barbera) [reply build.gradle:11]: I'll keep that in mind for the future. Thank you!

**Pull request closed without merging**